scripts/test-util.sh

```diff
@@ -42,6 +42,32 @@ checkout_repo() {
     fi
 }
 
+build_custom_linux() {
+    ARCH=$(uname -m)
+    SRCDIR=$PWD
+    LINUX_CUSTOM_DIR="$WORKLOADS_DIR/linux-custom"
+    LINUX_CUSTOM_BRANCH="ch-5.15.12"
+    LINUX_CUSTOM_URL="https://github.com/cloud-hypervisor/linux.git"
+
+    checkout_repo "$LINUX_CUSTOM_DIR" "$LINUX_CUSTOM_URL" "$LINUX_CUSTOM_BRANCH"
+
+    cp $SRCDIR/resources/linux-config-${ARCH} $LINUX_CUSTOM_DIR/.config
+
+    pushd $LINUX_CUSTOM_DIR
+    make -j `nproc`
+    if [ ${ARCH} == "x86_64" ]; then
+       cp vmlinux "$WORKLOADS_DIR/" || exit 1
+    elif [ ${ARCH} == "aarch64" ]; then
+       cp arch/arm64/boot/Image "$WORKLOADS_DIR/" || exit 1
+       cp arch/arm64/boot/Image.gz "$WORKLOADS_DIR/" || exit 1
+    fi
+    popd
+
+    if [ -d "$LINUX_CUSTOM_DIR" ]; then
+        rm -rf $LINUX_CUSTOM_DIR
```

Comment:
Won't this mean you always rebuild the kernel fully on source code / config change vs letting `make` decide to build or not if anything has actually changed.

Reply:
I think so, to be honest I am not sure why we have this for x86 and metrics script, probably because originally x86 and metrics scripts did not use `checkout_repo` function so that the source dir will not be auto-removed when branch/version is bumped. For Arm, this `rm -rf $LINUX_CUSTOM_DIR` is not needed.

I will remove this part in the PR, and done. (FYI, @sboeuf, as you also mentioned this part in https://github.com/cloud-hypervisor/cloud-hypervisor/pull/3892#discussion_r835051767)